web/libs/editor/src/tags/object/Paragraphs/model.js

```diff
@@ -403,6 +407,216 @@ const PlayableAndSyncable = types
       self.triggerSync("play");
       self.trackPlayingId();
     },
+
+    /**
+     * Seek to a specific phrase by index and select it
+     * @param {number} idx - The phrase index
+     */
+    seekToPhrase(idx) {
+      if (!isDefined(idx) || !self._value || idx < 0 || idx >= self._value.length) return;
+
+      const phrase = self._value[idx];
+      if (!isDefined(phrase.start)) return;
+
+      // If this phrase is already active, don't send seek messages (user might just be selecting text)
+      if (self.playingId === idx) {
+        return;
+      }
+
+      // Unselect regions that don't belong to the current phrase (enhanced annotation feature)
+      if (isFF(FF_LSDV_E_279)) {
```

Comment:
`seekToPhrase()` is only attached with FF on, so we should either attach it always to Phrase's `onClick` or remove FF checks inside the function.